### DISCUSSION TITLE
上周买的会员，过两天又变成不是会员了

### DISCUSSION THREAD

**Discussion by k7vza2** [getlantern/lantern-forum-cn#800] [General]
账户：k7vza2@163.com  麻烦管理调整一下 购买蓝灯已经几年了 从没遇过这样的事情
![Uploading 购买记录.jpg…]()

**Comment by k7vza2** [comment]
![购买记录](https://github.com/user-attachments/assets/67e72fa4-ab6f-43f6-8b46-fc7eb7339658)

**Comment by k7vza2** [comment]
有人能处理一下吗？

**Reply by MaxMKMax** [reply]
@k7vza2 您好. 已经帮您提交问题上去了.

**Comment by k7vza2** [comment]
已经恢复会员了 十分感谢帮忙！这两天用不了真的很着急，再次感谢！
